src/lib.rs

```diff
@@ -1,30 +1,52 @@
+/// # Fantom libhash-sha3
+///
+/// This crate is an sha3 implementation of the Hashing trait, defined in the libHash repository:
```

Comment:
You can use normal markdown:
```
[libhash](https://github.com/Fantom-foundation/libhash)
```
then check links by hand or use `cargo deadlinks --dir target/doc/libhash`, installed with `cargo install cargo-deadlinks`.